spec/spec_helper.rb

```diff
@@ -1,4 +1,6 @@
 require "codeclimate-test-reporter"
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.